source/MaterialXView/Window/WindowCocoaWrappers.h

```diff
@@ -0,0 +1,26 @@
+#ifndef MATERIALX_WINDOWCOCOAWRAPPERS_H
```

Comment:
As part of re-org. I've split out pure window Cocoa wrappers from GL specific ones.